File: app/src/main/java/com/purpletealabs/bitcoinapp/dtos/DefaultPriceListResponse.java
package com.purpletealabs.bitcoinapp.dtos;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class DefaultPriceListResponse {

    @SerializedName("bpi")
    @Expose
    private Bpi bpi;

    public Bpi getBpi() {
        return bpi;
    }
}

File: app/src/main/java/com/purpletealabs/bitcoinapp/dtos/DefaultPriceListResponse.kt
package com.purpletealabs.bitcoinapp.dtos

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class DefaultPriceListResponse {

    @SerializedName("bpi")
    @Expose
    val bpi: Bpi? = null
}
